lib/services/local/sessiontracker.go

```diff
@@ -160,6 +190,10 @@ func (s *sessionTracker) CreateSessionTracker(ctx context.Context, req *proto.Cr
 		HostUser:          req.HostUser,
 	}
 
+	if spec.Expires.IsZero() {
+		spec.Expires = now.Add(defaultSessionExpiry)
+	}
```

Comment:
This will conflict with https://github.com/gravitational/teleport/pull/11551, which treats `spec.Expires` as a static expiration if set, meaning `item.Expires` will be set to `spec.Expires` and won't be updated dynamically.

Since every existing code path will set `spec.Expires` to `24h` I think you can just remove this line and wait for https://github.com/gravitational/teleport/pull/11551 to be merged.

Reply:
👍